### PR TITLE
Added Code block

### PR DESCRIPTION
**Pull request by imsaptarshi**

### CODE REVIEW

File: core/package.json [hunk 1]
```diff
@@ -46,6 +46,9 @@
     "README.md"
   ],
   "dependencies": {
+    "clone-deep": "^4.0.1",
```
Comment (milan090) [inline core/package.json:49]: Why are there two clone-deep packages here?

File: core/src/components/Code/Code.component.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+};
+
+export const CodeComponent: React.FC<CodeProps> = ({ index, data }) => {
+    const { setBlockData } = useData();
+
+    const handleChange = (value:any) => {
+        setBlockData<string>(index, value);
+    };
+
+    /*const handleKeyDown = (e: React.KeyboardEvent<HTMLDivElement>): boolean | void => {
```
Comment (milan090) [inline core/src/components/Code/Code.component.tsx]: Remove unnecessary code

File: core/src/components/Code/Code.component.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import "ace-builds/src-noconflict/theme-monokai";
+
+interface CodeProps extends BlockComponentProps {
+    data: string;
+};
+
+export const CodeComponent: React.FC<CodeProps> = ({ index, data }) => {
+    const { setBlockData } = useData();
+
+    const handleChange = (value:any) => {
```
Comment (milan090) [inline core/src/components/Code/Code.component.tsx]: Why are you using any? Does the library not provide any types for this?

Comment (imsaptarshi) [reply core/src/components/Code/Code.component.tsx]: no :)

**Pull request merged**